src/Backend/views/site/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
View files in `src` isn't a good idea. What about `resources/backend/views` directory?